src/app/static/src/app/actions.ts

```diff
@@ -169,6 +173,35 @@ export const actions: ActionTree<RootState, RootState> = {
             context.dispatch(RootAction.FetchCostPerCaseGraphConfig),
             context.dispatch(RootAction.FetchCostTableConfig)
         ]);
+    },
+
+    [RootAction.DismissErrors](context) {
+        const {commit} = context;
+        commit(RootMutation.DismissErrors);
+    },
+
+    [RootAction.SetBudget](context, payload: { budget: number }) {
```

Comment:
Good point. I don't know why I thought that mutations should only be made from actions - especially as it's already done elsewhere! But I'll leave it as it is for now - at least until it's clear how the `projectOptions` stuff should work, just in case the action needs to do anything else.